packages/report-server/src/reportBuilder/transform/functions/position.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+type PositionParams = {
+  position?: string;
+};
+
+// Calculate which position to splice a new row into an array
+const positionAliases = {
+  before: (index: number, insertCount: number) => index + insertCount,
+  after: (index: number, insertCount: number) => index + insertCount + 1,
+  start: (index: number, insertCount: number) => insertCount,
```

Comment:
I don't know if `start` is necessarily useful, but it's basically free with the current implementation